README.md

```diff
@@ -5,3 +5,55 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 An [Apache Parquet](https://parquet.apache.org/) implementation in Rust (work in progress)
+
+## Development
+### Requirements
+- Rust nightly
+- Thrift 0.11+ or master (with Rust support)
+
+To install Rust nightly build, just run (assuming you have Rust installed already):
+```shell
+rustup update nightly
+rustup default nightly
+```
+
+Project requires Thrift with Rust support, which is in 0.11+/master. If release is not available on
+official website, follow instructions:
+```shell
+git clone --depth=1 https://github.com/apache/thrift
+# go to thrift project directory and install it
+cd thrift
+
+# export env vars for openssl
+LD_LIBRARY_PATH=/usr/local/opt/openssl/lib:"${LD_LIBRARY_PATH}"
+CPATH=/usr/local/opt/openssl/include:"${CPATH}"
+PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig:"${PKG_CONFIG_PATH}"
+export LD_LIBRARY_PATH CPATH PKG_CONFIG_PATH
+
+./bootstrap.sh
+# when configuring you might want to keep set of languages to a minimum, e.g. C++/Java/Rust
+./configure
```

Comment:
Perhaps we can add `enable-libs=no` here to avoid building all the language libs that we don't need?

Reply:
I just tried doing it, and it disables all libraries, e.g. `./configure --enable-libs=no --with-rust=yes`, all language options are no-op. I guees it is because of this https://github.com/apache/thrift/blob/master/configure.ac#L116. It looks like one would have to disable them manually. 

I updated README to include flags for no tests and tutorials. Also found that Python and PHP Thrift installations do not work (require additional privileges), but I did not mention it in README.

I can be wrong here, would appreciate if you could help with `enable-libs`.

Reply:
Hmm.. did you get any error with `./configure --enable-libs=no --with-rust=yes`?: I think it should be `with-rs=yes`. Also, I don't think we need to have `with-rs=yes` since `cargo build` will build this dependency (see [here](https://github.com/sunchao/parquet-rs/blob/master/Cargo.toml#L21)). This is also [how the CI does today](https://github.com/sunchao/parquet-rs/blob/master/.travis.yml#L18).

Reply:
I ran command `./configure --enable-libs=no --with-rs` and it listed libraries for each language - they all had "no" including Rust. Is this okay?

Reply:
I get something like this:
```
./configure --enable-libs=no --with-rs
...

thrift 1.0.0-dev

Building Plugin Support ...... : no
Building C++ Library ......... : no
Building C (GLib) Library .... : no
Building Java Library ........ : no
Building C# Library .......... : no
Building .NET Core Library ... : no
Building Python Library ...... : no
Building Ruby Library ........ : no
Building Haxe Library ........ : no
Building Haskell Library ..... : no
Building Perl Library ........ : no
Building PHP Library ......... : no
Building Dart Library ........ : no
Building Erlang Library ...... : no
Building Go Library .......... : no
Building D Library ........... : no
Building NodeJS Library ...... : no
Building Lua Library ......... : no
Building Rust Library ........ : no
```

Reply:
I am running `make install` to see what happens.

Reply:
Yes, you are right, just running `./configure --enable-libs=no` works! Thank you! I will update the doc.